micro-app/Zendesk ticket list/template-zdesk-ticket-list.json

```
@@ -1,24 +1,24 @@
 {
     "body": [
         {
-                "type": "ColumnSet",
-                "columns": [
-                    {
-                        "type": "Column",
-                        "width": "auto",
-                        "items": [
-                            {
-                                "type": "Image",
-                                "url": "https://w7.pngwing.com/pngs/813/178/png-transparent-logo-brand-zendesk-scalable-graphics-business-text-people-logo-thumbnail.png",
-                                "size": "Large"
-                            }
-                        ]
-                    }
-                ]
-            },
+            "type": "ColumnSet",
+            "columns": [
+                {
+                    "type": "Column",
+                    "width": "auto",
+                    "items": [
+                        {
+                            "type": "Image",
+                            "url": "https://cdn.icon-icons.com/icons2/2429/PNG/512/zendesk_logo_icon_147198.png",
+                            "size": "Small"
+                        }
+                    ]
+                }
+            ]
+        },
         {
             "type": "Container",
-            "$data": "${actions.ticket_list_api.response.body.tickets}",
+            "$data": "${actions.listtickets.response.body.results}",
```

Dans le README zendesk ticket list le nom du block API utilisé est "ticket_list_api" donc vu que tu laisses "ticket_list_api" dans le README, il vaut mieux le laisser ici aussi:
```suggestion
            "$data": "${actions.ticket_list_api.response.body.results}",
```
Sinon mettre listtickets à la place de ticket_list_api dans le README